spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/RetryableConfigServicePropertySourceLocator.java

```diff
@@ -0,0 +1,72 @@
+package org.springframework.cloud.config.client;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.core.env.CompositePropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.core.env.PropertySource;
+import org.springframework.retry.RetryCallback;
+import org.springframework.retry.RetryContext;
+import org.springframework.retry.backoff.BackOffPolicy;
+import org.springframework.retry.backoff.ExponentialBackOffPolicy;
+import org.springframework.retry.policy.NeverRetryPolicy;
+import org.springframework.retry.policy.SimpleRetryPolicy;
+import org.springframework.retry.support.RetryTemplate;
+
+public class RetryableConfigServicePropertySourceLocator extends
+		AbstractConfigServicePropertyLocator {
+
+	private static Log logger = LogFactory
+			.getLog(RetryableConfigServicePropertySourceLocator.class);
+
+	private RetryTemplate retryTemplate;
```

Comment:
Good catch, I corrected that as well as the `RestTemplate` inside `ConfigServicePropertySourceLocator`